app/scripts/components/exploration/components/datasets/dataset-list-item.tsx

```diff
@@ -367,51 +351,64 @@ function DatasetTrack(props: DatasetTrackProps) {
     });
   }, [xScaled, dataset]);
 
+  const { blocks, wasLumped } = lumpBlocks({
+    domain: domainToRender,
+    xScaled,
+    timeDensity: dataset.data.timeDensity
+  });
+
   return (
     <svg width={width} height={DATASET_TRACK_BLOCK_HEIGHT}>
-      {domainToRender.map((date) => (
-        <DatasetTrackBlock
-          key={date.getTime()}
-          xScaled={xScaled}
-          date={date}
-          dataset={dataset}
-          isVisible={isVisible}
-        />
+      {blocks.map(([blockStart, blockEnd]) => (
+        <g key={blockStart.getTime()}>
+          <DatasetTrackBlock
+            xScaled={xScaled}
+            startDate={blockStart}
+            endDate={blockEnd}
+            isVisible={isVisible}
+          />
+          {wasLumped && (
+            <DatasetGLabel
+              fill='white'
+              x={Math.max(xScaled(blockStart), 2) + 2}
+              y={12}
+            >
+              G
+            </DatasetGLabel>
+          )}
```

Comment:
Not a fan of the 'G' letter to signal a group. Is there a more visual way to express this? Like for instance:

- a "stacked" look
<img width="347" alt="Screenshot 2023-09-14 at 09 34 39" src="https://github.com/NASA-IMPACT/veda-ui/assets/1583415/481db66a-67bc-4872-8ca8-8a5e3b07262d">

- or an SVG pattern that reproduces what it already looks like when not lumped:
<img width="378" alt="Screenshot 2023-09-14 at 09 36 00" src="https://github.com/NASA-IMPACT/veda-ui/assets/1583415/0d256f41-b6a6-4afa-a580-0d2ab318e861">

Reply:
I like this "stacked" look suggestion a lot! CC/ @danielfdsilva

Reply:
This was a great suggestion!